posthog/models/entity/entity.py

```diff
@@ -86,9 +86,9 @@ def __init__(self, data: Dict[str, Any]) -> None:
 
         self._action: Optional[Action] = None
         self._data = data  # push data to instance object so mixins are handled properly
+
         if self.type == TREND_FILTER_TYPE_EVENTS and not self.name:
-            # It won't be an int if it's an event, but mypy...
-            self.name = str(self.id)
+            self.name = "All events" if self.id is None else str(self.id)
```

Comment:
We handle this client-side everywhere else I think (correct me if I'm wrong) so I'd maybe do that for this case too

Reply:
this used to just send the string "None", as `str(None) == 'None'`

Reply:
Hmm, in that case I'd say we should send `null`, no?

Reply:
In fairness this is `name` being adjusted and not `id`, so let's leave "All events" here 👍